examples/select_object_content.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# MinIO Python Library for Amazon S3 Compatible Cloud Storage, (C) 2019 MinIO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from minio import Minio
+from minio.error import ResponseError
+from minio import SelectObjectReader
+from minio.select_object_reader import CRCValidationError
+from minio.select_object_options import (SelectObjectOptions, CSVInput, JSONInput, RequestProgress, ParquetInput, InputSerialization, OutputSerialization, CSVOutput, JsonOutput)
+
+client = Minio('s3.amazonaws.com',
+               access_key='YOUR-ACCESSKEY',
+               secret_key='YOUR-SECRETKEY')
+
+obj = SelectObjectOptions(
+    expression=" select * from s3object",
+    input_serialization = InputSerialization(compression_type="NONE",
+                              csv=CSVInput(FileHeaderInfo="USE",
+                                           RecordDelimiter="\n",
+                                           FieldDelimiter=",",
+                                           QuoteCharacter='"',
+                                           QuoteEscapeCharacter='"',
+                                           Comments="#",
+                                           AllowQuotedRecordDelimiter="FALSE"
+                                           ),
+                             ),     
+                             # If input is JSON
+                             # json = JSONInput(Type="DOCUMENT"
+                             #                 )
+                             #         ),
+
+    output_serialization=OutputSerialization(
+                        csv = CSVOutput(
+                            QuoteFields="ASNEEDED",
+                            RecordDelimiter="\n",
+                            FieldDelimiter=",",
+                            QuoteCharacter='"',
+                            QuoteEscapeCharacter='"',
+                            )
+                        ),
+
+    # If output is JSON
+    # output_serialization=OutputSerialization(
+    #                 json = JsonOutput(
+    #                     RecordDelimiter="\n"
+    #                     )
+    #                 ),
+    request_progress=RequestProgress(
+        enabled="TRUE"
+        )
+    )
+
+try:
+    response_stream = client.select_object_content('<YOUR_BUCKET>', '<YOUR_OBJECT>', obj)
+    data = SelectObjectReader(response_stream)
```

Comment:
SelectObjectReader() should be inside the code in select_object_content - we shouldn't return a response stream.

Reply:
done.